packages/common/README.md

```diff
@@ -4,4 +4,13 @@
 
 Part of the [Temporal](https://temporal.io) [Node.js SDK](https://www.npmjs.com/package/temporalio).
 
-See documentation on the [documentation site](https://docs.temporal.io/docs/node/introduction).
+You should probably not be using this package directly. See instead:
```

Comment:
Some things are exported only from this package (maybe we should re-export them where relevant though).

Reply:
like what?